integration/basic_test.go

```diff
@@ -26,7 +26,7 @@ func (s *SimpleSuite) TestNoOrInexistentConfigShouldFail(c *check.C) {
 	time.Sleep(500 * time.Millisecond)
 	output := b.Bytes()
 
-	c.Assert(string(output), checker.Contains, "No configuration file found")
+	c.Assert(string(output), checker.Not(checker.Contains), "No configuration file found")
```

Comment:
I'm not sure to understand, 
Staert doesn't fail if no configuration file found.
We should check nothing is printed in `output`, WDYT @emilevauge ?